app/Filament/Resources/GameHashResource.php

```diff
@@ -39,19 +43,48 @@ public static function table(Table $table): Table
                     ->label('MD5')
                     ->searchable()
                     ->fontFamily(FontFamily::Mono)
-                    ->url(fn (GameHash $record): string => route('game.hash.manage', ['game' => $record->game])),
+                    ->url(fn (GameHash $record): string => route('game.hash.manage', ['game' => $record->game]))
+                    ->toggleable(isToggledHiddenByDefault: false),
 
                 Tables\Columns\TextColumn::make('game.title')
                     ->label('Game')
                     ->searchable()
-                    ->url(fn (GameHash $record): string => route('game.show', ['game' => $record->game])),
+                    ->url(function (GameHash $record) {
+                        if (request()->user()->can('manage', Game::class)) {
+                            return GameResource::getUrl('view', ['record' => $record->game]);
+                        }
+
+                        return route('game.show', ['game' => $record->game]);
+                    }),
+
+                Tables\Columns\TextColumn::make('game.system')
+                    ->label('System')
+                    ->formatStateUsing(fn (System $state) => "[{$state->id}] {$state->name}")
+                    ->url(function (GameHash $record) {
+                        if (request()->user()->can('manage', System::class)) {
+                            return SystemResource::getUrl('view', ['record' => $record->id]);
```

Comment:
The system link is incorrect. There aren't 40000+ systems.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/1df265b2-a5ad-45de-a192-b9880fbf8dab)

Reply:
Thanks - resolved in latest.